poc_iot_injector/src/server.rs

```diff
@@ -208,7 +216,7 @@ async fn handle_txn_submission(
                     "txn submission failed!, hash: {:?}",
                     txn_details.hash_b64_url
                 );
-                Err(Error::TxnSubmission(txn_details.hash_b64_url))
+                Err(HandleTxnSubmissionError(txn_details.hash_b64_url))
```

Comment:
I had a patch locally where I wasn't encapsulating this into its own error since it reduces visibility, let's just propogate the actual Error originating from the node follower on txn submission and just return `Err(Error:from(err))` instead?

Reply:
I'm not sure what you mean, this basically does the same thing as before

Reply:
It's fine. I can patch it in a smaller PR. Basically I would like to remove the `HandleTxnSubmissionError` entirely and just propagate the error from node_follower up.